example-monorepos/blank/apps/expo/package.json

```diff
@@ -1,28 +1,28 @@
 {
   "dependencies": {
     "app": "*",
-    "expo": "^44.0.0",
-    "expo-splash-screen": "~0.14.1",
-    "expo-status-bar": "~1.2.0",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
-    "react-native": "0.64.3",
-    "react-native-gesture-handler": "~2.1.0",
-    "react-native-reanimated": "~2.3.1",
-    "react-native-safe-area-context": "3.3.2",
-    "react-native-screens": "~3.10.1",
-    "react-native-web": "0.17.1"
+    "expo": "^46.0.0",
+    "expo-splash-screen": "~0.16.1",
+    "expo-status-bar": "~1.4.0",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
+    "react-native": "0.69.4",
+    "react-native-gesture-handler": "~2.5.0",
+    "react-native-reanimated": "~2.9.1",
+    "react-native-safe-area-context": "4.3.1",
+    "react-native-screens": "~3.15.0",
+    "react-native-web": "~0.18.7"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.9",
-    "@types/react": "~17.0.21",
-    "@types/react-native": "~0.64.12",
-    "typescript": "~4.3.5"
+    "@babel/core": "^7.18.6",
+    "@types/react": "~18.0.0",
+    "@types/react-native": "~0.69.1",
+    "typescript": "^4.6.3"
   },
   "scripts": {
     "start": "expo start --dev-client",
-    "android": "expo run:android",
-    "ios": "expo run:ios",
+    "android": "expo run:android -c",
```

Comment:
What's -c command ? Is this the ``--config`` 

<img width="923" alt="Screenshot 2022-09-08 at 11 32 46 AM" src="https://user-images.githubusercontent.com/107257599/189046280-cbc7eaa2-c56c-4eb5-bf2e-bd8b5be9c980.png">

Reply:
I know that for `expo start` this clears the cache, but as far as I know it does nothing for `expo run:ios`, so I'll remove it.